core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -269,6 +269,15 @@ trait AttributeComponent {
       }
     }
 
+    def findUniqueStringsByNameQuery(attrName: AttributeName, queryString: String) = {
+      filter(rec =>
+        rec.namespace === attrName.namespace &&
+        rec.name === attrName.name &&
+        rec.valueString.isDefined &&
+        rec.valueString.like(s"%$queryString%")
```

Comment:
I would not trust the slick logger, I would try to capture it on the mysql side.